Exec/RegTests/EB_BackwardStepFlame/input.2d

```diff
@@ -29,14 +29,13 @@ prob.T_wall = 300
 prob.meanFlowMag = 10.0
 prob.Y_fuel = 0.0445269
 prob.Y_oxid = 0.2226345
-
 #---------------------- PeleLM CONTROL ---------------------------
 peleLM.v = 1                                # PeleLMeX verbose
 peleLM.use_wbar = 1                         # Include Wbar term in species diffusion fluxes
 peleLM.sdc_iterMax = 2                      # Number of SDC iterations
 peleLM.num_init_iter = 3                    # Number of initial iterations
 peleLM.floor_species = 0                    # Enforce species positivity (non-conservative !)
-
+peleLM.do_patch_flow_variables = true
```

Comment:
set to false because this isn't actually used in the tutorial.

Reply:
change incorporated